docs/dev_guide/command-line-options-reference.rst

```diff
@@ -35,9 +35,8 @@ in alphabetical order.
    * - ``--build-script-file=<file>``
      - Specifies the name of generated makefile for migrated file(s). Default
        name: ``Makefile.dpct``.
-   * - ``=c_cxx_standard_library``
-     - A set of functions from the C and C++ standard libraries that are allowed
-       to be used in SYCL device code.
+   * - ``--change-cuda-files-extension-only``
+     - Limit extension change to .cu and .cuh files only. Default: off.

```

Comment:
Formatting for rendered docs: Add double back ticks (``) around file extensions (.cu, .cuh) and default value (off).